test/spec/Feature/Query/PlanSpec.hs

```diff
@@ -241,7 +242,7 @@ spec actualPgVersion = do
           resStatus  = simpleStatus r
 
       liftIO $ do
-        resHeaders `shouldSatisfy` elem ("Content-Type", "application/vnd.pgrst.plan+text; charset=utf-8")
+        resHeaders `shouldSatisfy` elem ("Content-Type", "application/vnd.pgrst.plan+text; for=\"application/json\"; charset=utf-8")
```

Comment:
Will move the plan media type changes to another PR. Overall this media type was hard to get right for this feature.

Reply:
Done in https://github.com/PostgREST/postgrest/pull/2833